test/cpp/nativert/CMakeLists.txt

```diff
@@ -3,7 +3,9 @@ set(NATIVERT_TEST_ROOT ${TORCH_ROOT}/test/cpp/nativert)
 # Build the cpp gtest binary containing the cpp-only tests.
 set(NATIVERT_TEST_SRCS
   ${NATIVERT_TEST_ROOT}/test_tensor_meta.cpp
+  ${NATIVERT_TEST_ROOT}/test_placement.cpp
   ${TORCH_ROOT}/torch/nativert/graph/TensorMeta.cpp
+  ${TORCH_ROOT}/torch/nativert/executor/Placement.cpp
```

Comment:
Seems like you are missing includes here?

Reply:
@yushangdi I don't think you need to include this here because you already marked Placement as TORCH_API so the test binary can find the implementation from libtorch directly.

Alternatively you can remove TORCH_API marker and still keep this source file here. But it's strange to have both TORCH_API and this being builtin into the test binary directly (which may cause ODR violation)

Reply:
> @yushangdi I don't think you need to include this here because you already marked Placement as TORCH_API so the test binary can find the implementation from libtorch directly.
> 
> Alternatively you can remove TORCH_API marker and still keep this source file here. But it's strange to have both TORCH_API and this being builtin into the test binary directly (which may cause ODR violation)

@zhxchen17 Sure. I removed the dependency.